src/viewer.js

```diff
@@ -48,22 +48,34 @@ async function startServer(bundleStats, opts) {
 
   if (!chartData) return;
 
-  const app = express();
-  app.use(express.static(`${projectRoot}/public`));
-
-  app.get('/', (req, res) => {
-    res.writeHead(200, {'Content-Type': 'text/html'});
-    const html = renderViewer({
-      mode: 'server',
-      title: resolveTitle(reportTitle),
-      chartData,
-      defaultSizes,
-      enableWebSocket: true
-    });
-    return res.end(html);
+  const sirvMiddleware = sirv(`${projectRoot}/public`, {
+    dev: true
```

Comment:
What does this flag mean? Let's add a comment about it please.

Reply:
It prevents caching for watch mode. Added comment.